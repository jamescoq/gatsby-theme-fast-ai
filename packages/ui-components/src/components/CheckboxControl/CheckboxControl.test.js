import React from 'react';
import { shallow } from 'enzyme';

import CheckboxControl from './CheckboxControl';

describe('<CheckboxControl />', () => {
	const requiredProps = { name: 'foo' };

	it('shallow', () => {
		const wrapper = shallow(<CheckboxControl {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
