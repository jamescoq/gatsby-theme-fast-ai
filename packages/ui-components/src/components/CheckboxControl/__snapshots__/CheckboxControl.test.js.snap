// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CheckboxControl /> shallow 1`] = `
<FormControl
  component="input"
  name="foo"
  onBlur={[Function]}
  onFocus={[Function]}
  type="checkbox"
/>
`;
